treee111/wahooMapsCreator
# [FEAT] correctly integrate tunnels to maps in macOS (in the same way as in Windows)

## This PR…

- closes #160
- transforms tunnels under macOS to be included in the generated maps
- adds tunnels to the VTM-theme to be shown under BOLT, ROAM and BOLTv2

## Considerations and implementations

During implementation, the `osmium` call for macOS was refactored to a `osmosis` call like used in Windows.
A lot of thoughts and tryouts for tunnels in macOS can be seen in the assigned issue.

- Two .osm.pbf files can be compared with [osmium diff](https://docs.osmcode.org/osmium/latest/osmium-diff.html) , example call: `osmium diff -c merged.osm.pbf merged_osmosis.osm.pbf`
- The [documentation](https://wiki.openstreetmap.org/wiki/Osmosis/Detailed_Usage_0.48#--merge_(--m)) for osmosis helped during refactoring.

## How to test

1. create .map files using `-c`
2. check the created mapfiles in cruiser with different `defaultvalue` in the VTM theme to check the different devices

## Pull Request Checklist
    + Especially the [Pull-Requests](https://github.com/treee111/wahooMapsCreator/blob/develop/.github/CONTRIBUTING.md#Pull-Requests) section
